svfs/fs.go

```diff
@@ -59,6 +59,8 @@ var (
 	// of flags. Each flag enables an optimization that can be used by storage
 	// synchronization processes in order to reduce network access.
 	TransferMode int
+	// Represents the time when svfs is mounted
```

Comment:
Honor go comments guidelines : it should start with the variable name then comes the comment about it. Run [golint](https://github.com/golang/lint) to make sure this is ok.